src/test/MavenTestDevModeActions.ts

```diff
@@ -129,8 +132,10 @@ it('start maven with options from liberty dashboard', async () => {
 it('start maven with history from liberty dashboard', async () => {  
 
   const reportPath = path.join(utils.getMvnProjectPath(),"target","site","failsafe-report.html");
-  const deleteReport = await utils.deleteReports(reportPath);
-  expect (deleteReport).to.be.true;  
+  const alternateReportPath = path.join(utils.getMvnProjectPath(), "target", "reports", "failsafe.html");
+  let deleteReport = await utils.deleteReports(reportPath);
+  let deleteAlternateReport = await utils.deleteReports(alternateReportPath);
+  expect (deleteReport || deleteAlternateReport).to.be.true;  
```

Comment:
Same as above, _both_ reports should be deleted or not exist at this point. Instead of an OR, this should be an AND.